### DISCUSSION TITLE
Access Denied

### DISCUSSION THREAD

**Discussion by thelumlaa** [Azure/ipam#114] [Q&A] [answered]
All of a sudden all of my admins lost access to IPAM, including the API calls via Service Principal that was given access to the Engine App. I streamed logs and any request from any admin just leads to the following error

`2023-04-13T13:25:32.578523803Z Message: Please provide below info when asking for support: timestamp = 2023-04-13T13:25:15.3176720Z, correlationId = 
2023-04-13T13:25:32.578530703Z Exception Details:	(AccessDenied) Access is denied to the requested resource. The user might not have enough permission.
2023-04-13T13:25:32.578537403Z 	Code: AccessDenied`

I don't recall changing anything, the access worked last week. All the enterprise app credentials are valid, tried to restart the app but it doesn't help. Any idea what I can do to fix it?
Thanks

**Comment by thelumlaa** [comment]
We can sign-in through SSO, we all can see the admins list in the UI - and the list is not changed, we all are still set as admins - but any request to blocks/vnets/subnets/reservation just infinitely spins and outputs the error above in the logs.

**Comment by DCMattyG** [comment, marked as answer]
Hi @thelumlaa, I responded to your issue #115. I'll continue to respond there so we keep a single thread.

We're here to help however we can 👍
